app/views/kits/_form.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= simple_form_for @kit, remote: request.xhr?, html: { class: 'form-horizontal' } do |f| %>
```

Comment:
I like the `remote: request.xhr?` -- that's cool :D

Reply:
@armahillo I copied this from the items form. :D